roles/idrac_gather_facts/README.md

```diff
@@ -119,7 +118,7 @@ ansible.utils
     <td>- System <br> - BIOS <br> - Controller <br> - CPU <br> - Enclosure <br> - EnclosureEMM <br> - Fan <br>
         - Firmware <br> - HostNIC <br> - License <br> - Memory <br> - NIC <br> - PCIeSSDBackPlane <br>
         - PowerSupply <br> - PresenceAndStatusSensor <br> - Sensors_Battery <br> - Sensors_Intrusion <br>
-        - Sensors_Voltage <br> - VirtualDisk <br> - PCIeDevice <br> - PhysicalDisk <br> - SystemMetrics</td>
+        - Sensors_Voltage <br> - VirtualDisk <br> - PCIeDevice <br> - PhysicalDisk <br> - SystemMetrics<br> - SecureBoot</td>
     <td>list</td>
     <td>- I(target) component for which information needs to be gathered.</td>
```

Comment:
can we update the component specific description as well here? (as in the bspecs)

Reply:
Sure